helpers/types.ts

```diff
@@ -20,3 +20,9 @@
 export function isDefined<T>(x: T | undefined | null): x is T {
   return x !== undefined && x !== null;
 }
+
+// Request types
+
+export type RequestOptions = {
+  onVersionChange?: VoidFunction;
+};
```

Comment:
I'm not sure it's the right file to put a type because it's inside the helper folder, so I'd say it should only contain helper functions...

I guess this type would be fine inside `init.ts`...